client/stylesheets/_toCat.scss

```diff
@@ -2414,7 +2414,8 @@ i:hover {
 
 .headerAuth {
     margin-right: 20px !important;
-    margin-top: 13px !important;
+    margin-bottom: 8px !important;
+    margin-top: 8px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces